androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/stopDetails/TripHeaderCardTest.kt

```diff
@@ -0,0 +1,431 @@
+            .assertIsDisplayed()
+
+        composeTestRule
+            .onNodeWithText(formatTime(predictionDeparture), useUnmergedTree = true)
+            .assertIsDisplayed()
+    }
+
+    @Test
+    fun testScheduled() {
+        val now = Clock.System.now()
+        val objects = ObjectCollectionBuilder()
+        val stop = objects.stop { name = "Stop Name" }
+
+        val scheduledTime = now.plus(5.minutes)
+        val route = objects.route {}
+
+        val schedule = objects.schedule { departureTime = scheduledTime }
+
+        composeTestRule.setContent {
+            TripHeaderCard(
+                "",
+                TripHeaderSpec.Scheduled(
+                    stop,
+                    TripDetailsStopList.Entry(
+                        stop = stop,
+                        stopSequence = 0,
+                        alert = null,
+                        schedule = schedule,
+                        prediction = null,
+                        vehicle = null,
+                        routes = listOf()
+                    )
+                ),
+                stop.id,
+                TripRouteAccents(route),
+                now
+            )
+        }
+
+        composeTestRule
+            .onNodeWithText("Scheduled to depart", useUnmergedTree = true)
+            .assertIsDisplayed()
+        composeTestRule.onNodeWithText(stop.name, useUnmergedTree = true).assertIsDisplayed()
+        composeTestRule
+            .onNodeWithText(formatTime(scheduledTime), useUnmergedTree = true)
+            .assertIsDisplayed()
+        composeTestRule
+            .onNodeWithText(formatTime(scheduledTime), useUnmergedTree = true)
+            .assertIsDisplayed()
+    }
+    /*
+    @Test
+    fun testScheduledTap()  {
+        val now = Clock.System.now()
+                val objects = ObjectCollectionBuilder()
+        val stop = objects.stop { _ in }
+
+        val schedule = objects.schedule { schedule in
+                schedule.departureTime = now.addingTimeInterval(5 * 60).toKotlinInstant()
+        }
+
+        val tapExpectation = expectation(description: "card tapped")
+
+        val sut = TripHeaderCard(
+                spec: .scheduled(stop, .init(
+        stop: stop,
+        stopSequence: 0,
+        alert: null,
+        schedule: schedule,
+        prediction: null,
+        vehicle: null,
+        routes: []
+        )),
+        tripId: "",
+        targetId: stop.id,
+        routeAccents: .init(),
+        onTap: { tapExpectation.fulfill() },
+        now: now
+        )
+
+        try sut.inspect().find(ViewType.ZStack.self).callOnTapGesture()
+            wait(for: [tapExpectation], timeout: 1)
+            try XCTAssertNotnull(sut.inspect().find(ViewType.Image.self, where: { image in
+                    try image.actualImage().name() == "fa-circle-info"
+                    }))
+            }
+            */
+
+    @Test
+    fun testAccessibilityVehicleDescriptionSelectedStop() {
```

Comment:
broke this test up a bit from what was on the iOS side, since the iOS test also included accessibility functionality not yet implemented for clickability.